sdk/client/resourcemanager/poller_lro.go

```diff
@@ -170,11 +169,37 @@ func (p *longRunningOperationPoller) Poll(ctx context.Context) (result *pollers.
 		for k, v := range statuses {
 			if strings.EqualFold(string(op.Properties.ProvisioningState), string(k)) {
 				result.Status = v
-				return
+				break
 			}
 			if strings.EqualFold(string(op.Status), string(k)) {
 				result.Status = v
-				return
+				break
+			}
+		}
+
+		if result.Status == pollers.PollingStatusFailed {
+			result.HttpResponse.Body = io.NopCloser(bytes.NewReader(respBody))
```

Comment:
rather than doing this here and on line 194, we can instead do this once above (line 119)

Reply:
Thanks!